manifests/g/gpodder/gpodder/3.10.21/gpodder.gpodder.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: gpodder.gpodder
+PackageVersion: 3.10.21
+PackageLocale: en-US
+Publisher: gpodder
+PackageName: gPodder

```

```suggestion
Publisher: The gPodder Team
PackageName: gPodder - Media aggregator and podcast client
```
These values must match Control Panel